### DISCUSSION TITLE
Response live streaming isn't working at all

### DISCUSSION THREAD

**Discussion by rcdailey** [danny-avila/LibreChat#4247] [Help Wanted]
I have librechat set up on a home server and I use it on three separate machines:

1. A macbook pro using MS Edge (personal)
2. A windows PC using MS Edge (personal)
3. A windows laptop using MS Edge (work)

1 & 2 both work the same and correctly. When I send a message to Anthropic or OpenAI models, I get the response streamed back to me in real time.

However, for number 3, the response is not streamed back to me for any model. Instead, I get the response all at once, and I get a waiting indicator until the response is presented.

I can only assume there's some work policy or maybe a firewall setting preventing this from working, but I have no way of really knowing what the issue is. I did check the MS Edge dev tools console for any errors but I don't see any.

Could you help me figure out why the 3rd scenario behaves differently?

**Comment by danny-avila** [comment]
MS Edge
https://github.com/user-attachments/assets/7da8430b-443a-47c0-99a9-e7a360181633

**Reply by rcdailey** [reply]
Yeah like I said, it's working on 2 machines for me as well. But it doesn't work on my work PC. How can I identify what would cause the issue on that one machine?

**Reply by rcdailey** [reply]
@danny-avila Let me know if there's any diagnostic steps I can run.
